modules/core/src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -613,6 +613,40 @@ object KafkaConsumer {
             .toMap
         }
 
+      override def offsetsForTimes(
+        timestampsToSearch: Map[TopicPartition, Long]
+      ): F[Map[TopicPartition, OffsetAndTimestamp]] =
+        withConsumer.blocking {
+          _.offsetsForTimes(
+              timestampsToSearch.asJava.asInstanceOf[util.Map[TopicPartition, java.lang.Long]]
+            )
+            // Filter empty/missing partition null values for more idiomatic scala
+            .toMapNoNullValues
```

Comment:
Perhaps it should return a `Map[TopicPartition, Option[OffsetAndTimestamp]]` instead of filtering out the null entries?

Reply:
I can change it to that if that is desired.

Reply:
Updated to `Option` rather than removing the nulls